### PR TITLE
SSR-3317: Implement custom css for Table

### PR DESCRIPTION
**Pull request by voduchau**

### CODE REVIEW

File: docs/document/docs/components/desktop/table.md [hunk 1]
```diff
@@ -58,6 +58,22 @@ Here is a list of available constructors:
...
+Please check [Custom CSS feature guide](../../getting-started/custom-css.md) at first.
+:::
+
+Here is a list of properties that can be used for modifying component style:
+#### Property
+| Name | Description |
+| :--- | :--- |
+| --kuc-table-header-cell-background-color | |
+| --kuc-table-header-cell-{index}-width | <li>This property allows you to set the width of specific table columns based on their index values. For instance, you can define the width of the first column using `--kuc-table-header-cell-0-width`</li><li>Please note that the index values start from `0`, with `0` corresponding to the first column</li> |
+| --kuc-table-header-cell-width | <li>This property is used to set the width for all columns in a table. It defines a uniform width for every column</li><li>If you need to set a specific width for an individual column, you can use the `--kuc-table-header-cell-{index}-width` property</li> |
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/table.md:72]: Please unify not to use commas in the sentence.

Comment (TomokoMiyake) [reply docs/document/docs/components/desktop/table.md:72]: Did you decide not to include `--kuc-table-header-cell-width: auto` setting Tips as we discussed here?
https://bozuman.cybozu.com/k/23175/show#record=3317&comment=76

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]
Please check user guide review result.

File: docs/document/docs/components/desktop/table.md (hunk 1 above)
Comment (voduchau) [reply docs/document/docs/components/desktop/table.md:72]: >Please unify not to use commas in the sentence.

I'm sorry, I don't understand what you mean here.
Is it that I have to rewrite the descriptions for the properties without using commas (`,`) in the sentence?

>Did you decide not to include --kuc-table-header-cell-width: auto setting Tips as we discussed here?

I will add it to the `Description` column after we decide on the custom CSS name.
>>When you want to add the element that will expand/contract as a child component, you might be able to keep the behavior setting `--kuc-table-header-cell-width: auto`.

Comment (TomokoMiyake) [reply docs/document/docs/components/desktop/table.md:72]: > Is it that I have to rewrite the descriptions for the properties without using commas (,) in the sentence?

I made a mistake! Sorry. I mean period (`.`) not comma.

> I will add it to the Description column after we decide on the custom CSS name.

Thank you!

> When you want to add the element that will expand/contract as a child component, you might be able to keep the behavior setting --kuc-table-header-cell-width: auto.

You can modify it like this.
When you want to add the element that will expand/contract as a child component, you might be able to keep the behavior by setting `--kuc-table-header-width: auto`.

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]
Please check the review comment.

File: docs/document/docs/components/desktop/table.md (hunk 1 above)
Comment (voduchau) [reply docs/document/docs/components/desktop/table.md]: >You can modify it like this.

I updated it.
https://github.com/kintone-labs/kintone-ui-component/pull/1422/commits/af1dced6727c0a922375f86fd3fea52d29330a19

>I made a mistake! Sorry. I mean period (.) not comma.

You mean the period at the end of the sentence ?
I'm sorry, but I currently don't see any period (.) at the end of the sentence. Please help me double-check it.
Thank you!

Comment (TomokoMiyake) [reply docs/document/docs/components/desktop/table.md]: There is a period here: "in s table.".
It's a small point and not important, but I'm pointing out to unify the format in user guide.

```
<li>This property is used to set the width for all columns in a table. It defines a uniform width for every column</li>
```

Comment (voduchau) [reply docs/document/docs/components/desktop/table.md]: I updated it
https://github.com/kintone-labs/kintone-ui-component/pull/1422/commits/75bf57a92486914b6f4b86997f2cec1964f665f0

**Review by TomokoMiyake — APPROVED** [review]
LGTM!

**Pull request merged**